vendor/engines/c2po/spec/services/purchase_order_account_builder_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe PurchaseOrderAccountBuilder, type: :service do
```

Comment:
I think this file is missing tests. I think it probably wants `it_behaves_like "AccountBuilder#build"`